### PR TITLE
[AC-661] Adding more details in unable to login dialog box

### PR DESCRIPTION
**Pull request by Devanshk9**
## Description of what I changed
The unable to login button shows little info and it should show more contact information that can guide the user if he is not able to log in and also provide support email.

**### Screenshot of Emulator output**
Screenshot_1579344347

## Issue I worked on
JIRA Issue: https://issues.openmrs.org/browse/AC-661

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by Devanshk9** [comment]
Hi I added the details in the dialog box but not able to create the hyperlink can anyone help?

**Comment by ribhavsharma** [comment]
Hi devansh! Check [this](https://stackoverflow.com/questions/9290651/make-a-hyperlink-textview-in-android) out.
Also, could you edit the xml so that all of the text has the same margin from the left?

**Comment by Devanshk9** [comment]
> Hi devansh! Check [this](https://stackoverflow.com/questions/9290651/make-a-hyperlink-textview-in-android) out.
> Also, could you edit the xml so that all of the text has the same margin from the left?

Sure thanks @ribhavsharma I will do it soon.

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -33,13 +33,19 @@
...
+        Username- admin\n
+        Password- Admin123\n
+    </string>
     <string name="forgot_button_ok">Dismiss</string>
     <string name="login_offline">Offline</string>
     <string name="login_online">Online</string>
 
     <string name="emptyerror">Required</string>
     <string name="fname_empty_error">Given name required</string>
+    <string name="fmname_invalid_error">Given and middle name cannot contain symbols or numbers</string>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/strings.xml]: This isn't necessary. Please pull the latest changes from master.

Comment (Devanshk9) [reply openmrs-client/src/main/res/values/strings.xml]: Ok sure sir i will do that

**Comment by Devanshk9** [comment]
> @Devanshk9 Kindly wait for #681 to be merged, as the 'Unable to Login' text should link to that screen instead.

Ok sir

**Comment by rishabh-997** [comment]
I think this will be a redundant PR, the `unable to login` button will now start the contact us activity as discussed here rather than detailed dialogue box

https://github.com/openmrs/openmrs-contrib-android-client/pull/710

**Comment by VibhorChinda** [comment]
So we should close this PR.

**Comment by f4ww4z** [comment]
Yes this is true. Thanks @VibhorChinda and @rishabh-997 for the reminder, and @Devanshk9 for contributing.

**Pull request closed without merging**